src/main/java/com/exadel/discount/service/impl/CategoryServiceImpl.java

```diff
@@ -30,6 +31,21 @@ public CategoryDTO save(CategoryDTO categoryDTO) {
         return category;
     }
 
+    @Override
+    @Transactional
+    public CategoryDTO updateCategoryById(CategoryDTO categoryDTO, UUID id) {
+        log.debug(String.format("Update Category with ID %s", id));
+        return categoryRepository.findCategoryById(id)
```

Comment:
instead of getting entity return boolean from DB, ask @el-dub for details. 
The same for the vendor